src/calculateRentalCost.js

```diff
@@ -28,7 +28,27 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const rentalCost = dayCost * days;
+  let result = 0;
+  const discount = 20;
+  const discountPlus = 50;
+
+  if (days < 3) {
+    result = rentalCost;
+
+    return result;
```

Comment:
```suggestion
    return rentalCost;
```
Just return value immediately